flathub.json

```diff
@@ -0,0 +1,3 @@
+{
+    "only-arches": ["x86_64", "i386"]
```

Comment:
Does it fail to build on arm and aarch64?

Reply:
I can't try it on arm or aarch64, so I don't know if it is working well there. But, as long as all the libs are also available on these platform, then my software *should* work too :)

Reply:
Let's try it then; we use it only when repackaging ready binaries or CI fails for unknown reasons. We can always disable it here too.

Reply:
Yeah - I love making sure it works first - but it's simple to find out if it works on ARM platforms and fix the most common errors.

Reply:
:+1: